app/src/main/java/com/monkeyteam/chimpagne/ui/event/polls/CreatePollDialog.kt

```diff
@@ -58,7 +58,7 @@ fun CreatePollDialog(
                   text = stringResource(id = R.string.chimpagne_confirm),
                   modifier = Modifier.testTag("confirm poll button"),
                   onClick = {
-                    if (title == "" || query == "" || options.contains("")) {
+                    if (title.isEmpty() || query.isEmpty() || options.contains("")) {

```

Comment:
I like the refactoring you do, they are appreciated as much as in your reviews B)

Reply:
still unsure of the usefulness of this